### ISSUE TITLE
Have to re-start gui before I can chat when following new user

### ISSUE THREAD

**Issue by dorfsmay** [keybase/client#5831]
If I try start to follow a user, then decide to chat with them, I can find them in the gui, but it goes back to the empty chat gui when I click on "start a chat". The only way I can start a chat with them at this point, is to completely shutdown keybase, and re-run run_keybase.

my log id: bd334dcbb37c2fb6670e9d1c

**Comment by maxtaco** [comment]
cc: @keybase/react-hackers

**Comment by gabriel** [comment]
Yeah I think if you don't send a message after first creating a chat, it gets lost on next inbox refresh. 

I created internal issue here: https://keybase.atlassian.net/browse/DESKTOP-2837
Maybe prioritize? (It was surprising to me too.)
